bedrock/settings/base.py

```diff
@@ -126,8 +227,7 @@ def JINJA_CONFIG():
     path('media'),
 )
 
-JINGO_EXCLUDE_APPS = (
-    'admin', 'registration', 'rest_framework', 'rna', 'waffle')
+JINGO_EXCLUDE_APPS = ('admin', 'registration', 'rest_framework', 'rna', 'waffle')
```

Comment:
I would prefer a list that evokes the pristine beauty of a Hawaiian waterfall. Please change it.